docs/UserGuide.md

```diff
@@ -68,36 +70,36 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 Shows a message explaning how to access the help page.
 
+**TODO**: Change help message image
 ![help message](images/helpMessage.png)
 
 Format: `help`
 
 
 ### Adding a property: `add`
 
-Adds a property to the address book.
+Adds a property to PropertyWhiz.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `add n/PROPERTY_NAME a/PROPERTY_ADDRESS s/SELLER_NAME p/SELLER_PHONE $/PRICE_MIN [t/TAG]…​` e.g., `add n/Blk 123 a/123, Clementi Rd, #04-20, 1234665 s/James Lee sp/61234567 $/100000 max/100000 t/HDB t/3rm`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A property can have any number of tags (including 0)
 </div>
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add n/Blk 123 a/123, Clementi Rd, #04-20, 1234665 s/James Lee sp/61234567 $/100000 max/100000 t/HDB t/3rm`
 
 ### Listing all properties : `list`
 
-Shows a list of all properties in the address book.
+Shows a list of all properties in PropertyWhiz.
 
 Format: `list`
 
 ### Editing a property : `edit`
 
-Edits an existing property in the address book.
+Edits an existing property in PropertyWhiz.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+Format: `edit INDEX [n/PROPERTY_NAME] [a/PROPERTY_ADDRESS] [s/SELLER_NAME] [p/SELLER_PHONE] [$/PRICE_MIN] [t/TAG]…​`
```

Comment:
Hmm.. I would prefer if this PR only translate the original UG to markdown, and then you can update the documentation of the tags in your better tags PR (because this PR is for v1.1 and better tags is for v1.2).

Reply:
Agree with @lizchow, take into account that we are already 1 day past our internal deadline for milestone 1.1

Reply:
Ok, I will resolve all these changes.